votingpool/pool.go

```diff
@@ -121,11 +161,11 @@ func LoadAndGetDepositScript(namespace walletdb.Namespace, m *waddrmgr.Manager,
 func LoadAndCreateSeries(namespace walletdb.Namespace, m *waddrmgr.Manager, version uint32,
 	poolID string, seriesID, reqSigs uint32, rawPubKeys []string) error {
 	pid := []byte(poolID)
-	vp, err := Load(namespace, m, pid)
+	p, err := Load(namespace, m, pid)
 	if err != nil {
-		managerErr := err.(waddrmgr.ManagerError)
-		if managerErr.ErrorCode == waddrmgr.ErrVotingPoolNotExists {
-			vp, err = Create(namespace, m, pid)
+		vpErr := err.(Error)
```

Comment:
As previously mentioned, this will panic if the error from Load is not a `votinpool.Error`.  Need to be sure that it only ever returns that.